addons/sale_timesheet/views/project_task_views.xml

```diff
@@ -38,6 +38,64 @@
                             <field name="currency_id" column_invisible="True"/>
                             <field name="cost_currency_id" column_invisible="True"/>
                         </tree>
+                        <kanban class="o_kanban_mobile">
+                            <field name="employee_id"/>
+                            <field name="sale_line_id"/>
+                            <field name="existing_employee_ids"/>
+                            <field name="partner_id"/>
+                            <field name="currency_id"/>
+                            <templates>
+                                <t t-name="kanban-box">
+                                    <div t-attf-class="oe_kanban_card oe_kanban_global_click">
+                                        <div class="row">
+                                            <div class="col-8 d-flex">
+                                                <field name="employee_id" widget="many2one_avatar_employee" context="{'active_test': True}"/>
+                                                <strong><span class="ps-1"><t t-esc="record.employee_id.value"/></span></strong>
```

Comment:
I think t-esc is deprecated in favor of t-out now (and I think it escapes by default unless it's a markup object)

Reply:
@reth-odoo 
Actually, `t-esc` is deprecated in QWeb backend (report, portal templates) but apparently it is not deprecated for JS templates. 🤷🏼‍♂️

Reply:
@reth-odoo @xavierbol kept the t-esc :shrug: 
Thanks for the quick review, check out my changes